web/pom.xml

```diff
@@ -875,10 +841,211 @@
     </plugins>
   </build>
 
-  <!-- Define profiles for running configuration using -Denv=profile.id parameter
-    Profiles configuration is defined in src/main/filters.
-    -->
   <profiles>
+
+    <!-- Schema management profiles. -->
+    <profile>
+      <id>iso19115-3.2018</id>
+      <activation>
+        <property><name>!copy</name></property>
+        <file><exists>../schemas/iso19115-3.2018</exists></file>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>${project.groupId}</groupId>
+          <artifactId>schema-iso19115-3.2018</artifactId>
+          <version>${gn.schemas.version.range}</version>
+        </dependency>
+      </dependencies>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-dependency-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>iso19115-3.2018-resources</id>
+                <phase>process-resources</phase>
+                <goals><goal>unpack</goal></goals>
+                <configuration>
+                  <encoding>UTF-8</encoding>
+                  <artifactItems>
+                    <artifactItem>
+                      <groupId>${project.groupId}</groupId>
+                      <artifactId>schema-iso19115-3.2018</artifactId>
+                      <version>${gn.schemas.version}</version>
+                      <type>jar</type>
+                      <overWrite>false</overWrite>
+                      <outputDirectory>${basedir}/src/main/webapp/WEB-INF/data/config/schema_plugins</outputDirectory>
+                      <includes>plugin/**</includes>
+                      <fileMappers>
+                        <fileMapper implementation="org.codehaus.plexus.components.io.filemappers.RegExpFileMapper">
+                          <pattern>^plugin/</pattern><replacement>./</replacement>
+                        </fileMapper>
+                      </fileMappers>
+                    </artifactItem>
+                  </artifactItems>
+                </configuration>
+              </execution>
+            </executions>
+          </plugin>
+        </plugins>
+      </build>
+    </profile>
+
+    <!-- Unpack schema resources from maven artifacts -->
+    <profile>
+      <id>unpack_schemas</id>
+      <activation> <!-- default, disabled by -Dcopy -->
+        <property><name>!copy</name></property>
```

Comment:
I noticed that if I run mvn -Pcopy-schemas then it will run both the copy-schemas (because I mentioned it in the profile) and the unpack_schemas (because -Dcopy does not exists).  The same issues probably happens when using the -P to set env variables.   I quickly looked online and did see others with similar issues but I did not see a easy fix - so this may not be easy to fix.  Just wanted to bring it to your attention as you may know of a way to fix this.

Reply:
Are you confusing `-D` and `-P`?

You can force a profile to run using `-P<profile>` (or not run using `-P!<profile>`. 

I think this is a case where in your IDE you are manually enabling/disabling some profiles rather than passing in `-Dcopy=true` as intended.

Reply:
I would like to experiment if we can manage `-Dcopy=true` in InteilJ and have it correctly activate (and deactivate) the two alternative profiles.

If not we are stuck:
- Adding `-Pcopy-schema -P!unpack-schema` to the run/debug configuration. 
- Enabling/disabling profiles in IntilliJ maven tool window

Reference: [work with maven profiles](https://www.jetbrains.com/help/idea/work-with-maven-profiles.html) IntelliJ

Reply:
Enabling/disabiling profiles in maven tool window worked.